src/main/groovy/edu/oregonstate/mist/locations/jsonapi/ResultObject.groovy

```diff
@@ -0,0 +1,9 @@
+package edu.oregonstate.mist.locations.jsonapi
+
+class ResultObject {
```

Comment:
Yes, I was going to originally put this in the skeleton, but I found: http://katharsis.io/ . If this framework doesn't work, then the classes inside the locations/jsonapi package will be moved over to the skeleton in a future sprint.